himin/day2/가운데 글자 가져오기.js

```diff
@@ -0,0 +1,12 @@
+function solution(s) {
+    let middleIndexS = Math.floor(s.length / 2)
+    let middleChar = "";
+      
+      if(s.length % 2 == 1) {
+         middleChar = s[middleIndexS];
+      } else {
+         middleChar = s[middleIndexS - 1] + s[middleIndexS]
+      }
+      
+      return middleChar;
+  }
```

Comment:
```suggestion
function solution(s) {
    const middleIndexS = Math.floor(s.length / 2);
    let middleChar = "";
      
    if (s.length % 2 === 1) {
       middleChar = s[middleIndexS];
    } else {
       middleChar = s[middleIndexS - 1] + s[middleIndexS];
    }
    
    return middleChar;
  }
```
@yihimin  값이 변하지 않는 변수는 `const`로 선언해주세요. 항상 "변수는 const로 우선 정의하자!" 가 1순위가 되어야 합니다 ㅎㅎㅎ 그러다가 재할당이 필요하다면 let으로 변경해주는 흐름이 안전한 코드 작성하는데 도움이 됩니다.

`줄맞추기`, `공백`, `세미콜론` 컨벤션 부분에서 아직은 미흡한데 제출 전에 3번 검토하면 꼭하시면 좋을 것 같습니다 :)

Reply:
498f1d69

@ssi02014 명심하겠습니다. 좋은 습관 잡아주셔서 감사합니다^^